internal/integration/publish_test.go

```diff
@@ -15,35 +15,56 @@
 package integration
 
 import (
-	"bytes"
 	"context"
 	"encoding/json"
-	"net/http"
+	"sort"
+	"strings"
 	"testing"
 	"time"
 
+	"github.com/golang/protobuf/proto"
 	authorizedappmodel "github.com/google/exposure-notifications-server/internal/authorizedapp/model"
+	exportapi "github.com/google/exposure-notifications-server/internal/export"
 	exportdatabase "github.com/google/exposure-notifications-server/internal/export/database"
 	exportmodel "github.com/google/exposure-notifications-server/internal/export/model"
+	"github.com/google/exposure-notifications-server/internal/pb/export"
 	publishdb "github.com/google/exposure-notifications-server/internal/publish/database"
 	publishmodel "github.com/google/exposure-notifications-server/internal/publish/model"
+	"github.com/google/exposure-notifications-server/internal/serverenv"
+	"github.com/google/exposure-notifications-server/internal/storage"
 	"github.com/google/exposure-notifications-server/internal/util"
 	verifyapi "github.com/google/exposure-notifications-server/pkg/api/v1alpha1"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
 )
 
 func TestPublish(t *testing.T) {
 	t.Parallel()
 
 	ctx := context.Background()
-	env, client := testServer(t)
+
+	// Export
+	exportConfig := &exportapi.Config{
+		CreateTimeout:  10 * time.Second,
+		WorkerTimeout:  10 * time.Second,
+		MinRecords:     1,
+		PaddingRange:   1,
+		MaxRecords:     10000,
+		TruncateWindow: 1 * time.Millisecond,
+		MinWindowAge:   1 * time.Second,
+		TTL:            336 * time.Hour,
+	}
+
+	env, client := testServer(t, exportConfig)
```

Comment:
The testServer will be used across all test cases and we can't predict all customizations required for each particular test. Approach with passing the config allows us to be more flexible. If it makes sense we can use default config in parameter.

Reply:
I think we should handle those customizations as-needed instead of doing it proactively. There are 7 different configs and it will quickly become unwieldy to support passing in configurations for all of them.

Reply:
I made this change because I already had to change the "default" export configuration for my specific use case. I agree that we don't want to pass all the configuration for now, but we'll probably have to do something like this in the future. This is just the first step.

Reply:
I think the default setup was just incomplete. I'd rather edit it inline until we have a real need to customize it.